server/src/internalClusterTest/java/org/elasticsearch/get/GetActionIT.java

```diff
@@ -824,11 +825,8 @@ void indexSingleDocumentWithStringFieldsGeneratedFromText(boolean stored, boolea
     }
 
     public void testGetRemoteIndex() {
-        IllegalArgumentException iae = expectThrows(IllegalArgumentException.class, () -> client().prepareGet("cluster:index", "id").get());
-        assertEquals(
-            "Cross-cluster calls are not supported in this context but remote indices were requested: [cluster:index]",
-            iae.getMessage()
-        );
+        IndexNotFoundException iae = expectThrows(IndexNotFoundException.class, () -> client().prepareGet("cluster:index", "id").get());
+        assertEquals("no such index [cluster:index]", iae.getMessage());
```

Comment:
I suspect this is fundamentally the same as Albert's comment, but I don't think we should change the behaviour here. Cross-cluster GET doesn't really make sense, and it doesn't look like we're implementing cross-cluster GET anyway so it seems strange to report `no such index` here. What if the remote index _does_ exist?